modules/channel-twilio/src/renderers/carousel.ts

```diff
@@ -0,0 +1,54 @@
+import * as sdk from 'botpress/sdk'
+import { ChannelRenderer } from 'common/channel'
+import { formatUrl } from 'common/url'
+import { CHANNEL_NAME } from '../backend/constants'
+import { TwilioContext } from '../backend/typings'
+
+export class TwilioCarouselRenderer implements ChannelRenderer<TwilioContext> {
+  get channel(): string {
+    return CHANNEL_NAME
+  }
+
+  get priority(): number {
+    return 0
+  }
+
+  get id(): string {
+    return TwilioCarouselRenderer.name
+  }
+
+  handles(context: TwilioContext): boolean {
+    return !!context.payload.items?.length
+  }
+
+  render(context: TwilioContext) {
+    const payload = context.payload as sdk.CarouselContent
+
+    for (const { subtitle, title, image, actions } of payload.items) {
+      const body = `${title}\n\n${subtitle || ''}`
+      const options: sdk.ChoiceOption[] = []
+
+      for (const button of actions || []) {
+        const title = button.title as string
+
+        if (button.action === 'Open URL') {
```

Comment:
i can't believe we still use this string everywhere. 
I'd love if we could use an enum or at least decent typings for this, it's so error prone